### PR TITLE
feat: update onboard to v2, removed v1

### PR DESCRIPTION
**Pull request by georgesmithishere**
https://www.notion.so/eb96722b397d42c9908002ca06af88e3?v=1f519ba1849347aa9740faedda4acf3b&p=58e036feb34c45b890079dea31ccc78d

### CODE REVIEW

**Comment by r0x4xn** [comment]
App <strong>core-dapp</strong> has been deployed to Vercel.

<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://core-dapp-71dv9v6hs-prepo.vercel.app'>https://core-dapp-71dv9v6hs-prepo.vercel.app</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="prj_JWVJ3WoV71UcOmB52qrcezjyBL3V"></hidden>

File: packages/prepo-utils/src/formatNumber.ts [hunk 1]
```diff
@@ -12,3 +13,5 @@ export const formatNumber: FormatNumber = (number, options): string => {
   if (number === undefined) return ''
   return new Intl.NumberFormat('en-US', numberFormatOptions).format(Number(number))
 }
+
+export const chainIdToHexString = (chainId: ChainId): string => `0x${chainId.toString(16)}`
```
Comment (chrisling-dev) [inline packages/prepo-utils/src/formatNumber.ts:17]: Could we move this to it's own file `chainIdToHexString.ts`? I think we've previously discussed to keep each util function in it's own file with it's named after itself so it's more organized

File: packages/prepo-stores/src/utils/onboard-config.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { InitOptions } from '@web3-onboard/core'
+import injectedModule from '@web3-onboard/injected-wallets'
+import { ChainId } from 'prepo-constants'
+import { getNetworkByChainId } from 'prepo-utils'
+
+const injected = injectedModule()
+
+export const chainIdToHexString = (chainId: ChainId): string => `0x${chainId.toString(16)}`
```
Comment (chrisling-dev) [inline packages/prepo-stores/src/utils/onboard-config.ts:8]: this is repeated in in prepo-utils, could just use the one in prepo-utils?

**Review by chrisling-dev — CHANGES_REQUESTED** [review]
There seems to be a bug when you disconnect and connect again, it will show:
Screenshot 2022-06-20 at 11 59 28 AM

**Comment by georgesmithishere** [comment]
> chainIdToHexString

fixed, thanks

File: packages/prepo-stores/src/utils/onboard-config.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      },
+    ],
+    appMetadata: {
+      name: appName,
+      icon: IconString,
+      description: 'Welcome to prePO',
+      recommendedInjectedWallets: [
+        { name: 'MetaMask', url: 'https://metamask.io' },
+        { name: 'Coinbase', url: 'https://wallet.coinbase.com/' },
+      ],
```
Comment (r0x4xn) [inline packages/prepo-stores/src/utils/onboard-config.ts:31]: We used to have support for more wallets. Can we add them here too?

**Review by r0x4xn — CHANGES_REQUESTED** [review]
Same behavior as Chris. After disconnecting wallet and try to connect again it get's a bit weird. 
Are we really cleaning the state of everything when we disconnect?

File: packages/prepo-stores/src/utils/onboard-config.ts (hunk 3 above)
Comment (georgesmithishere) [reply packages/prepo-stores/src/utils/onboard-config.ts:31]: we support huge amount of wallets, this is what injected wallets do, it tracks all installed wallets you have,
here is recommendation when you don't have any wallets installed - do we want to add more wallet here?

**Review by r0x4xn — APPROVED** [review]

**Pull request merged**